dali/pipeline/executor/workspace_policy.h

```diff
@@ -112,7 +112,7 @@ void add_output(op_type_to_workspace_t<op_type> &ws, const tensor_data_store_que
 template <OpType op_type>
 void SetupInputOutput(op_type_to_workspace_t<op_type> &ws, const OpGraph &graph, const OpNode &node,
                       const std::vector<tensor_data_store_queue_t> &tensor_to_store_queue,
-                      const QueueIdxs idxs) {
+                      const QueueIdxs &idxs) {
```

Comment:
These are merely three integers. If anything, I'd remove the `const` but pass it by value.
@klecki What do you think?

Reply:
The `const` is here to indicate that it's an input parameter, and the lack of reference was most probably intentional due to the size of the object.

Reply:
It stands out, though. We don't use that convention anywhere else.